docs/install.md

```diff
@@ -18,7 +18,7 @@ The instructions here are for Linux and MacOS. For installation on Windows, see
     ```bash
     virtualenv -p python3 venv
     source venv/bin/activate
-    pip3 install -r requirements.txt
+    pip3 install --upgrade .
```

Comment:
Could you explain to me why you are modifying that?

The previous line "pip3 install -r requirements.txt" was doing: "Install the requirements specified in this file".
https://pip.pypa.io/en/stable/user_guide/#requirements-files

Semantically, what we want, is to make sure we have the correct versions for the packages within "requirements.txt".

The new line is doing something different. What was you goal? Does it guarantee the requirements do match "requirements.txt"?

Reply:
I see, it's because the package is expected to be installed.